PyStemmusScope/global_data/cci_landcover.py

```diff
@@ -1,12 +1,16 @@
 """Module for loading and validating the ESA CCI land cover dataset."""
 from pathlib import Path
 from typing import Union
+import dask
 import numpy as np
 import pandas as pd
 import xarray as xr
 from PyStemmusScope.global_data import utils
 
 
+# see https://docs.dask.org/en/latest/array-slicing.html#efficiency
+dask.config.set(**{"array.slicing.split_large_chunks": True})  # type: ignore
```

Comment:
I am not sure if this needs to be repeated in every file. Shouldn't it be sufficient to add this to the global_data_selection.py file? That module imports all these separate datasets.

Or possibly a better solution would be to use the context manager in this file:
https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/76264fe638af3c4094973e73820f8ea32b92545e/PyStemmusScope/forcing_io.py#L133

And change that to:
```py
with dask.config.set(**{"array.slicing.split_large_chunks": True}):
    return global_data.collect_datasets(
        global_data_dir=global_data_dir,
        latlon=(lat, lon),
        time_range=(start_time, end_time),
        timestep=timestep,
    )
```

Reply:
Yes, it is sufficient to add it only to `focing_io`. I added it to each because those modules can be called outside the `forcing_io`. But I think your suggestion is fine too.